src/sparseml/sparsification/modifier_lr.py

```diff
@@ -23,6 +23,7 @@
     BaseUpdate,
     ModifierProp,
 )
+from sparseml.sparsification.types import SparsificationTypes
```

Comment:
nit: For consistency

```suggestion
from sparseml.sparsification import SparsificationTypes
```

Reply:
since this is in the same package, we run the risk of a circular dependency being introduce in the future if we inherit from the parent init that we export to